app/models/service_template_ansible_tower.rb

```diff
@@ -24,7 +24,7 @@ def self.create_catalog_item(options, _auth_user = nil)
 
   def remove_invalid_resource
     # remove the resource from both memory and table
-    service_resources.to_a.delete_if { |r| r.destroy unless r.reload.resource.present? }
+    service_resources.to_a.delete_if { |r| r.destroy if r.resource.blank? }
```

Comment:
@jrafanie you're okay with just ```service_resources.to_a.delete_if { |r| r.destroy }``` if that works, right? Can you please put a comment here just so I can have all the feedback in one place?

Reply:
Yeah, I don't understand the use case for needing it.  If the tests pass, I'm of the opinion we should remove the `.reload` or the former `.resource(true)` until we better understand why it's needed and can write tests showing this use case.